docs/src/man/split_apply_combine.md

```diff
@@ -60,31 +66,27 @@ We show several examples of the `by` function applied to the `iris` dataset belo
 ```jldoctest sac
 julia> using DataFrames, CSV, Statistics
 
-julia> iris = DataFrame(CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")));
-
-julia> first(iris, 6)
-6×6 DataFrame
-│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species     │ id    │
-│     │ Float64     │ Float64    │ Float64     │ Float64    │ String      │ Int64 │
-├─────┼─────────────┼────────────┼─────────────┼────────────┼─────────────┼───────┤
-│ 1   │ 5.1         │ 3.5        │ 1.4         │ 0.2        │ Iris-setosa │ 1     │
-│ 2   │ 4.9         │ 3.0        │ 1.4         │ 0.2        │ Iris-setosa │ 2     │
-│ 3   │ 4.7         │ 3.2        │ 1.3         │ 0.2        │ Iris-setosa │ 3     │
-│ 4   │ 4.6         │ 3.1        │ 1.5         │ 0.2        │ Iris-setosa │ 4     │
-│ 5   │ 5.0         │ 3.6        │ 1.4         │ 0.2        │ Iris-setosa │ 5     │
-│ 6   │ 5.4         │ 3.9        │ 1.7         │ 0.4        │ Iris-setosa │ 6     │
-
-julia> last(iris, 6)
-6×6 DataFrame
-│ Row │ SepalLength │ SepalWidth │ PetalLength │ PetalWidth │ Species        │ id    │
-│     │ Float64     │ Float64    │ Float64     │ Float64    │ String         │ Int64 │
-├─────┼─────────────┼────────────┼─────────────┼────────────┼────────────────┼───────┤
-│ 1   │ 6.7         │ 3.3        │ 5.7         │ 2.5        │ Iris-virginica │ 145   │
-│ 2   │ 6.7         │ 3.0        │ 5.2         │ 2.3        │ Iris-virginica │ 146   │
-│ 3   │ 6.3         │ 2.5        │ 5.0         │ 1.9        │ Iris-virginica │ 147   │
-│ 4   │ 6.5         │ 3.0        │ 5.2         │ 2.0        │ Iris-virginica │ 148   │
-│ 5   │ 6.2         │ 3.4        │ 5.4         │ 2.3        │ Iris-virginica │ 149   │
-│ 6   │ 5.9         │ 3.0        │ 5.1         │ 1.8        │ Iris-virginica │ 150   │
+julia> iris = DataFrame(CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")))
```

Comment:
I think we used `first` and `last` because we couldn't find a way to reproduce the default output in doctests. Unless you have an idea about how to fix this, probably better not make it even harder to enable doctests one day.

Reply:
I would leave this as is (and maybe disable doctests for selected chunks if this is not fixable). My idea is that this is a manual, so we should show things that one would normally do. And normally you just show the data frame and rely on default behaviour I think.

@mortenpi - do you know if it is possible to set a custom width and height of `stdout` in doctests?

Reply:
As you like. Anyway enabling doctests will take some work.

Reply:
If I understand the question correctly, then it is an open issue in Documenter, but there might a workaround via setting the appropriate environment variables (`COLUMNS` and [maybe `LINES`?](https://stackoverflow.com/questions/1780483/lines-and-columns-environmental-variables-lost-in-a-script)): https://github.com/JuliaDocs/Documenter.jl/issues/942#issuecomment-597905221

Reply:
Thank you! (you understand the question correctly :smile:)

@nalimilan - so we are safe, and when we switch to enable doctests the PR (or a standard practice) in Documenter.jl probably will be merged.